cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/HourlyCloudPricingDTO.java

```diff
@@ -0,0 +1,34 @@
+package com.cloudesire.platform.apiclient.dto.model.dto;
+
+import io.swagger.annotations.ApiModel;
+
+import java.math.BigDecimal;
+import java.util.Map;
+
+@ApiModel( "Configure pricing according to order minimum duration" )
+public class HourlyCloudPricingDTO extends BaseEntityDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2160)